oauthproxy.go

```diff
@@ -364,7 +364,11 @@ func (p *OAuthProxy) SaveSession(rw http.ResponseWriter, req *http.Request, s *s
 // RobotsTxt disallows scraping pages from the OAuthProxy
 func (p *OAuthProxy) RobotsTxt(rw http.ResponseWriter) {
 	rw.WriteHeader(http.StatusOK)
-	fmt.Fprintf(rw, "User-agent: *\nDisallow: /")
+	_, err := fmt.Fprintf(rw, "User-agent: *\nDisallow: /")
+	if err != nil {
+		logger.Printf("Error writing robots.txt: %s", err.Error())
+		p.ErrorPage(rw, 500, "Internal Error", err.Error())
```

Comment:
I was unsure about this one -- is ErrorPage still fine on this after `WriteHeader` & `Fprintf` potentially modified `rw`

Reply:
Ooh that is a good question, one to which I have no idea 😅 
I guess we can try to write the error page, though it's likely to fail. No idea how we would test this 🤔 
The alternative is to just set the response code and log that there was an error writing to it?